components/purchase_modal/purchase.scss

```diff
@@ -298,24 +149,156 @@
+                    padding-left: 24px;
+                    border: 1px solid #dadfe5;
+                    background-color: var(--center-channel-bg);
+                    border-radius: 0 0 4px 4px;
+
+                    .plan_action_btn {
+                        width: 100%;
+                        height: 40px;
+                        padding: 10px 24px;
+                        border: none;
+                        background: none;
+                        border-radius: 4px;
+                        color: #1c58d9;
+                        font-size: 14px;
+                        font-weight: 700;
+                        line-height: 14px;
+
+                        &.grayed {
+                            background: rgba(63, 67, 80, 0.08);
+                            color: rgba(63, 67, 80, 0.32);
+                        }
+
+                        &.active {
+                            border: 1px solid #1c58d9;
+                        }
+
+                        &.special {
+                            background: #1c58d9;
+                            color: #fff;
+                        }
+                    }
+
+                    .plan_price_rate_section {
+                        width: 100%;
+                        height: 150px;
+                        border-bottom: 1px solid #dadfe5;
+                        text-align: center;
+
+                        h4 {
+                            color: rgba(33, 63, 107, 0.8);
+                            font-family: 'Metropolis';
+                            font-size: 20px;
+                            font-style: normal;
+                            font-weight: 400;
+                            letter-spacing: -0.02em;
+                            line-height: 30px;
+                        }
+
+                        h1 {
+                            color: #1e325c;
+                            font-family: 'Metropolis';
+                            font-size: 52px;
+                            font-style: normal;
+                            font-weight: 700;
+                            letter-spacing: -0.02em;
+                            line-height: 60px;
+                        }
+
+                        .enterprise_price {
+                            font-size: 32px;
+                        }
+
+                        p {
+                            color: #152234;
+                            font-family: 'Metropolis';
+                            font-size: 18px;
+                            font-style: normal;
+                            font-weight: 400;
+                            letter-spacing: -0.02em;
+                            line-height: 20px;
+                        }
+                    }
+
+                    .plan_payment_commencement {
+                        margin-top: 16px;
+                        margin-bottom: 24px;
+                        color: #3f4350;
+                        font-family: 'Open Sans';
```

Comment:
Can you check if we can omit most of the `font-family` rules? Usually in webapp and cws, we have two distinct font uses, one for headlines and  regular text . I'm guessing the default font-family most elements inherit is already `Open Sans`, and for the titles that are `h1` and `h4`, they may also already be inheriting the correct family. In that case, overriding to the already inherited value does not change the value applied, and makes it harder to update to new fonts in the future.